bin-src/lib/getOptions.ts

```diff
@@ -75,6 +78,7 @@ export default function getOptions({ argv, env, flags, log, packageJson }: Conte
     branchName,
     patchHeadRef,
     patchBaseRef,
+    ...extraOptions,
```

Comment:
@ghengeveld the main issue with your approach is then the checks below (e.g. for missing project token) start hassling me if I pass the token in via options (rather than flags). We should have a chat about how we want this to work.